package.json

```diff
@@ -29,10 +29,9 @@
   "author": "Gregor Martynus",
   "license": "Apache-2.0",
   "devDependencies": {
-    "ava": "^4.3.3",
-    "c8": "^7.12.0",
+    "@types/node": "^22.15.17",
+    "c8": "^10.1.3",
```

Comment:
I fear the version bump of c8 makes the CI fail as we still test in Node 16. I'm okay bumping minimal version to Node 20 as 18 will be out of maintenance soon. Either as part of this PR and release a breaking change, or do that separately with a follow up PR

Reply:
let me fix that